test/functional/wallet_groups.py

```diff
@@ -90,8 +90,8 @@ def run_test(self):
         # ~0.1 and 1.4 and should come from the same destination
         values = [vout["value"] for vout in tx3["vout"]]
         values.sort()
-        assert_approx(values[0], 0.1, 0.0001)
-        assert_approx(values[1], 1.4)
+        assert_approx(values[0], vexp=0.1, vspan=0.0001)
+        assert_approx(values[1], vexp=1.4, vspan=0.0001)
```

Comment:
while here, there are six other `assert_approx` assertions without keyword args (two also in the new test), and two of the others which could optionally also have an explicit default vspan added

Reply:
Fair point. Fixed all of them.